action_server/frontend/src/lib/formData.ts

```diff
@@ -141,6 +142,24 @@ export const propertiesToFormData = (
   return entries;
 };
 
+/**
+ * Sets the title for one item of a list (i.e.: each item of a list
+ * should not have the same title as the list itself so that it's
+ * clearer what's an item of a list vs the list itself).
+ * 
+ * @param item The item which should have the title set.
+ */
+export const setArrayItemTitle = (item: PropertyFormData) => {
```

Comment:
This is an additional fix to list items so it's clearer what's the `list` item and what's the `list (item)`.

Final UI:

![image](https://github.com/robocorp/robocorp/assets/117621/06149dea-3c29-43f1-b660-7295368a94b5)